app/catalog/catalog_cli.go

```diff
@@ -156,6 +157,37 @@ var catalogCmdDef = &cli.Command{
 				util.CmdMiddlewareTracingSpan,
 			),
 		},
+		{
+			Name:  "serve",
+			Usage: "Generates html and serves",
+			Action: util.ChainCmdMiddleware(cmdServe,
+				util.CmdMiddlewareLogging,
+				util.CmdMiddlewareTracingConfig,
+				util.CmdMiddlewareTracingSpan,
+			),
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:    "output",
+					Aliases: []string{"o"},
+					Usage:   "Output path for HTML generation",
+				},
```

Comment:
Might prefer to get rid of this, per other comments (below) about making this as works-by-default as possible.  We can just pick a path under `{rootWorkspaceControlDir}/tmp/` (or `$WARPFORGE_RUNPATH`?)